source/documentation/appendix-docs/botor.md

```diff
@@ -204,7 +244,7 @@ replace `s3tools` calls.
 #### `read_using`
 
 ```r
-read_using <- function(FUN, s3_path, overwrite = TRUE, ...) {
```

Comment:
Why are we removing `overwrite` as an arg?

Reply:
It's not in the s3tools arguments and it's not needed for reading into a dataframe.